x/bsonx/bsoncore/value_test.go

```diff
@@ -619,10 +634,19 @@ func TestValue(t *testing.T) {
 			nil,
 			[]interface{}{primitive.NewDecimal128(12345, 67890), true},
 		},
+		{
+			"Timestamp.String/Success", Value.String, Value{Type: bsontype.Timestamp, Data: AppendTimestamp(nil, 12345, 67890)},
+			nil,
+			[]interface{}{"{\"$timestamp\":{\"t\":12345,\"i\":67890}}"},
```

Comment:
For clarity, what was the result of this before the change to `value.go`?

Reply:
```
value_test.go:673: return values at index 0 are not equal. got {"$timestamp":{"t":"12345","i":"67890"}}; want {"$timestamp":{"t":12345,"i":67890}}
```